README.md

```diff
@@ -23,35 +23,40 @@ $ sequelize [--HARMONY-FLAGS]
 
 
 ```
-Sequelize [CLI: v0.0.4, ORM: v1.7.5]
+Sequelize [Node: 0.10.36, CLI: 1.7.0, ORM: 2.0.2]
 
 
 Usage
   sequelize [task]
 
 Available tasks
-  db:migrate        Run pending migrations.
-  db:migrate:undo   Revert the last migration run.
-  help              Display this help text. Aliases: h
-  init              Initializes the project.
-  init:config       Initializes the configuration.
-  init:migrations   Initializes the migrations.
-  init:models       Initializes the models.
-  migration:create  Generates a new migration file. Aliases: migration:generate
-  version           Prints the version number. Aliases: v
+  db:migrate             Run pending migrations.
+  db:migrate:old_schema  Update legacy migration table
+  db:migrate:undo        Revert the last migration run.
+  db:migrate:undo:all    Revert all migrations ran.
```

Comment:
Well it heavily depends on the use cases. While I was working for companies that invested quite some time in proper migrations which where perfectly undoable at any given point in time, I also saw the strategy of using one-way migrations only.

Anyways, this feature is useful in development for sure. 

Thanks for the heads up about the typo. I sort of think it makes the way it is but maybe I'm wrong. Feel free to provide a PR which changes it here: https://github.com/sequelize/cli/blob/master/lib/tasks/db.js#L65